### PR TITLE
Allow run-time control of the animator

### PR DESCRIPTION
**Pull request by mrozycki**
Co-authored-by: pnwkw <pnwkw@users.noreply.github.com>

### CODE REVIEW

File: animator/src/controller.rs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+            loop {
+                animation = match rx.try_recv() {
+                    Ok(name) => Self::new_animation(name.as_str(), &points),
+                    Err(mpsc::TryRecvError::Empty) => animation,
+                    _ => return,
+                };
+
+                match client.display_frame(&animation.frame(t)).await {
+                    Err(LightClientError::ConnectionLost) => return,
```
Comment (krzmaz) [inline animator/src/controller.rs:47]: nit: some log would probably be useful here

File: animator/src/controller.rs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    _ => (),
+                };
+
+                tokio::time::sleep(Duration::from_millis(33)).await;
+                t += 0.033;
```
Comment (krzmaz) [inline animator/src/controller.rs:52]: nit: maybe extract the step value to a constant and use `Duration::from_secs_f32()`?

File: animator/src/controller.rs (hunk 1 above)
Comment (mrozycki) [reply animator/src/controller.rs:47]: Logs aren't set up in the animator.

Comment (mrozycki) [reply animator/src/controller.rs:47]: I'll set them up in a separate PR.

File: animator/src/controller.rs (hunk 2 above)
Comment (mrozycki) [reply animator/src/controller.rs]: Done

**Pull request merged**